triangle-calculator.cpp

```diff
@@ -12,16 +12,22 @@ void radianDegrees(double& ansReference, double calcOutput) {
 
 void error(int error, string x) {
     switch (error) {
-        case 1:
-            cout << "\n\n" << "Error: Invalid angle input" << "\n";
-            cout << "Please enter valid angle's (must add to 180 degree's)." << "\n\n";
-        case 2:
-            cout << "Error: " << x << " is incalculabe with information provided." << "\n\n";
+    case 1:

```

Comment:
I'd suggest indenting the case's I do not know if this is necerssary but it is easier to read and understand.
As a question why have you used a break; is this to stop it from checking the other cases, I have not used switch statements much and so do not know if they continue to check other cases after running code in a matching case?

Reply:
You made the right assumption ! 
The other cases would still run... Which was kind of redundant at that point & with my "updates" since case 1 was being triggered before the sides were inputted it was displaying an error message that shouldn't be displayed.